app/entrypoint.sh

```diff
@@ -79,6 +92,7 @@ source /app/functions.sh
 
 if [[ "$*" == "/bin/bash /app/start.sh" ]]; then
     check_docker_socket
+    get_nginx_gen_cid
```

Comment:
I think this should be: 
```
[[ -z "${NGINX_DOCKER_GEN_CONTAINER:-}" ]] && get_nginx_gen_cid
``` 
to allow still using a manual environment variable when you do have access to a static name.